src/Polly.Core/Simmy/Latency/LatencyStrategyOptions.cs

```diff
@@ -13,7 +13,7 @@ public class LatencyStrategyOptions : MonkeyStrategyOptions
     /// <remarks>
```

Comment:
We should also initialize the `Name` property for all chaos options. (same as for other strategies)

For example:

- `Chaos.Latency`
- `Chaos.Behavior`
- `Chaos.Outcome`
- `Chaos.Fault`

Reply:
Done